install

```diff
@@ -481,14 +481,14 @@ if !mac?
   #{Tty.bold}Fedora, Red Hat, CentOS, etc.#{Tty.reset}
     sudo yum groupinstall 'Development Tools'
   See #{Tty.underline}http://linuxbrew.sh/#dependencies#{Tty.reset} for more information.
-- Add Linuxbrew to your #{Tty.underline}#{shell_profile}#{Tty.reset} by running
+- Add the brew command to your #{Tty.underline}#{shell_profile}#{Tty.reset} by running
     echo 'eval $(#{HOMEBREW_PREFIX}/bin/brew shellenv)' >>#{shell_profile}
+  After modifying your shell profile, you may need to restart your session
+  (logout and then log back in) if the brew command isn't found.
```

Comment:
Remove this text.

Reply:
Michka moved this from below. Are you suggesting this text is just removed?

Reply:
Yes, I believe it should be removed and replaced with instructions telling the user to run these two shell commands:
```sh
echo 'eval $(#{HOMEBREW_PREFIX}/bin/brew shellenv)' >>#{shell_profile}
eval $(#{HOMEBREW_PREFIX}/bin/brew shellenv)
```